python/ray/_raylet.pyx

```diff
@@ -606,6 +607,10 @@ cdef store_task_errors(
         failure_object = RayTaskError(function_name, backtrace,
                                       exc, proctitle=proctitle,
                                       actor_repr=actor_repr)
+
+    # Pass the failure object back to the CoreWorker.
+    task_execution_error[0] = str(failure_object)
```

Comment:
Should we set the limit in the length of the string? Maybe we can do some truncation (if it is bigger than 500 chars or sth)